Added Documentation to OCA Explorer (#40)
bcgov/aries-oca-bundles: README.md

expect that other formats, such as W3C Verifiable Credentials Data Model
Standard JSON-LD credentials will eventually also be supported.

## Previewing And Designing OCA Bundles

When it comes to creating an OCA Bundle previewing the branding can be
done with the [OCA
Explorer](https://bcgov.github.io/aries-oca-bundles/)

### What is OCA Explorer

Overlays Capture Architecture (OCA) allows issuers to control the look
of a credentials in a users wallet. In addition, OCA makes supporting
multiple languages such as French and English easy.

### Using OCA Explorer

To design and preview an OCA Bundle, head over to the [OCA

> Explorer](https://bcgov.github.io/aries-oca-bundles/) and select an

Reviewer: ```suggestion
Explorer](https://bcgov.github.io/aries-oca-bundles/) and either select
```

> and either choose from an existing bundle in the repository or upload

Reviewer: ```suggestion
an existing bundle in the repository or upload
```

> a new one.

Reviewer: ```suggestion
one of your own.
```

![Select OCA Bundle](./docs/assets/upload_OCA_bundle.PNG)

> To view a preview of the selected or uploaded OCA Bundle, simply

Reviewer: ```suggestion

```

> select it and the preview will appear, allowing you to choose from a

Reviewer: ```suggestion
Once you have selected or uploaded an OCA Bundle, a preview will display. If configured in the bundler there may be the option to choose from a
```

variety of available languages using radio buttons based on the
[multilingual overlay](https://github.com/swcurran/aries-rfcs/tree/oca4aries/features/0755-oca-for-aries#oca-specification-overlays).

> The remaining fields are derived from the [branding

Reviewer: ```suggestion
The remaining fields displayed are derived from the [branding
```
